src/commands/lightning/dev/component.ts

```diff
@@ -79,12 +78,6 @@ export default class LightningDevComponent extends SfCommand<void> {
       }
 
       name = match.name;
-    } else {
```

Comment:
This should not be removed.... In the previous release we added the ability to use prompts and ask the user for values to different option flags. This applied to `app`, `site`, and `component` commands and was part of the user experience.

Basically the user should be able to just run `sf lightning dev` and interactively be asked a series of questions to determine what the user wants to do. Of course the user has the option of running `sf lightning dev blah blah` and provide the flag values but if they choose not to provide anything and just run `sf lightning dev` then the interactive experience should take over.